src/targets/gpu/adjust_allocation.cpp

```diff
@@ -0,0 +1,33 @@
+#include <migraphx/gpu/adjust_allocation.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/program.hpp>
+#include <migraphx/iterator_for.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace gpu {
+
+void adjust_allocation::apply(program& p) const
+{
+    for(auto ins : iterator_for(p))
+    {
+        if(ins->name() == "load")
```

Comment:
Maybe add `if (ins->inputs.empty()) continue` to skip instructions with no inputs.

Reply:
fixed